calcom/cal.com
# fix: Currency Not Reflecting Correctly #13845

## What does this PR do?

Fixes #13845 

Before 
as mentioned in the issue.

After
https://www.loom.com/share/5ea4fa237e10453da42007aef8919ef0?sid=be6495b3-5520-4ac2-9728-63802b649238

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
To test changes in this PR, make a paid event with currency JPY and try to book a time slot and see if amount is getting 100x or not. 


## Checklist


- I haven't checked if my changes generate no new warnings
- I haven't checked if existing unit tests pass locally with my changes
